### ISSUE TITLE
Deploy a `dev` instance of our app

### ISSUE THREAD

**Issue by fyliu** [hackforla/peopledepot#330]
### Dependency

 - #327

### Overview

The last part of our initial deployment is to deploy a `dev` instance for our own testing. ask ops to test and deploy either a `dev` instance that's triggered by updates to a branch such as `main`

### Action Items

- [x] ask ops to test and deploy a `dev` instance that's triggered by updates to a branch such as `main`
- [x] if it's easier, split this issue into manual deploy + auto deploy steps

### After Merge

- [ ] check off this issue in #297 and close it if done

### Instructions

- [ ] reference other projects' deployment GHAs to create one for ourselves

**Comment by ale210** [comment]
Hello, I have deployed the app to aws. It is available at: 

https://peopledepot-dev.vrms.io/

Some notes

- committing to the `main` branch will push code to the dev environment
- migrations are run at system startup, which will happen if you merge code
- Logs are available at: [here](https://us-west-2.console.aws.amazon.com/cloudwatch/home?region=us-west-2#logsV2:log-groups/log-group/$252Fecs$252Fpeople-depot-backend-dev)
- Database credentials are at: [here](https://us-west-2.console.aws.amazon.com/systems-manager/parameters/?region=us-west-2&tab=Table#list_parameter_filters=Name:Contains:people-depot)


@fyliu try to log into the incubator AWS account and confirm you have permissions for the above linked resources.


So things you might need to do

- return HTTP 200 from a request to the root url ('/'). If you don't it can anger the load balancer and pollute your logs with 400 errors and make it difficult to find information.
- there are a few missing resources that I see that may need to be figured out by the devs, like static resources

**Comment by fyliu** [comment]
@ale210 Thank you for getting this deployed and working. I can see the https://peopledepot-dev.vrms.io/api/v1/users/ rendering out.

1. There's a missing permission:
> * Database credentials are at: [here](https://us-west-2.console.aws.amazon.com/systems-manager/parameters/?region=us-west-2&tab=Table#list_parameter_filters=Name:Contains:people-depot)

I see this message when I try to view the page:
![Image](https://github.com/user-attachments/assets/f3b055d2-5fff-422d-b7f0-260d3604dfb0)

2. 
> * return HTTP 200 from a request to the root url ('/'). If you don't it can anger the load balancer and pollute your logs with 400 errors and make it difficult to find information.

Will make it return a simple page for now.

3. 
> * there are a few missing resources that I see that may need to be figured out by the devs, like static resources

We'll likely need to add some environment variables for this
```
STATIC_ROOT
STATIC_URL
STATICFILES_STORAGE
```

4. There's another issue #287 being worked on that's going to lead to adding more environment variables for the production config. Is this something I can add or do you need to add them? We'll have an example .env file with values to use for production like what we currently have for local development.

**Comment by ale210** [comment]
@fyliu I'll take a look at your permissions - can you confirm that your aws username is 'fangyiliu'? assuming it is but have to verify before we update permissions.

Regarding any additional environment variables, after this week you will be able to submit pull requests to incubator to add any vars that you need

**Comment by fyliu** [comment]
@ale210 yes, that's my username

**Comment by ale210** [comment]
@fyliu please try again thank you.

FYI, the username for the db owner is `people-depot_backend_dev_owner`, I don't believe that is in there

**Comment by fyliu** [comment]
@ale210 the permission is working. Thanks!

Can I merge the PR and work on the other items after? I could do them myself but would like to ask this Thursday if another developer wants to try them.

**Comment by ale210** [comment]
@fyliu yes you can merge this. This is really a first step to the additional changes
